documentation/MeetingMinutes/2024/MOM-2024-02-14.md

```diff
@@ -0,0 +1,134 @@
+    - **offline access** #123 https://github.com/camaraproject/IdentityAndConsentManagement/pull/123
+    - **Discussion on Issue #100 "Claims to be used in oAuth Access tokens".**
+    - **Discussion on Issue #108 "Question to documentation update for security schemes".**
+    - **Discussion on Issue #104 "Alignment of Client Credential authentication / authorization using API Gateway Pattern".**
+
+3. Action Points
+4. AoB
+
+<br>
+
+## Meeting Technicallities
+
+Jesús offered that ownership of the meeting invite could be transferred to the new chair. Axel said that for now this can stay as it is.
+
+
+## Discussion on Consensus in Camara Working Groups
+
+Recapitulation of the previous discussion on this topic: Jesús and Diego said there was consensus achieved during a WG call that we continue working on https://github.com/camaraproject/IdentityAndConsentManagement/pull/113
+Herbert said that lazy-consensus is the Camara way. Shilpa said no consensus was achieved. 
+Axel proposed that consensus should be achieved on Github because not everybody can attend meetings. Diego said that is not the Camara way.
+Shilpa proposed that if there is an important decision suggested to be made at the next meeting, then this should be communicated a few days before the meeting. So, that people know about the importance of the next meeting and attend when otherwise they would not attend.
+We agreed that is must be clear whether consensus is achieved or not, and that we need to get better on this.
+
+## Discussion the Way Forward regarding the Profile
+
+Jesús recapitulated the last meeting where PR #113 was discussed and that consensus was achieved.
+Axel and Shilpa did not agree with consensus beeing achieved - see previous section.
+Axel said that he created the issue https://github.com/camaraproject/IdentityAndConsentManagement/issues/122 and kindly asked WG members to comment on the issue.
+Axel also created PR https://github.com/camaraproject/IdentityAndConsentManagement/pull/121 and asked people to review it.
+
+After some discussion we agreed that WG members comment on the issue and review the PR seeking resolution and a decision whether to use the DT proposal or the TEF proposal as our WG's policy document.
+Elizabeth suggested that if discussions on GitHub get too long, we would call for a side meeting, anounce it on the mailing this and seek consensus.
+
+Axel agreed with this proposal and no objections where heard.
+
+Axel asked Bjorn Hjelm for his/OIDF's opinion. Bjorn said that OIDF is here to help.
+
+We agreed that if neither the commenting on the issue nor the review of the PR lead to consensus, the WG seek a vote at TSC.
+
+
+## Discussion about offline-access
+
+Jesús said that this PR "just" moves documentation from some place (GSMA) to another and that this WG agreed on moving documentation to Camara as the "place of truth" for both GSMA and Camara.
```

Comment:
```suggestion
Jesús said that as previously discussed and agreed in the WG, and as per CAMARA TSC (Technical Steering Committee) decision, all documentation related to the NBI CAMARA API interface will only be placed in CAMARA. CAMARA will be the source of truth for northbound interface. GSMA Opengateway will simply adopt what is defined in CAMARA. And Jesús suggests that we first focus on moving the necessary documentation to the CAMARA Identity & Consent WG repository as agreed. And then, if the WG participants want to revisit the content of the documentation, we can open a new issue for that, no problem. This would allow us to wrap up a long discussion, close issue https://github.com/camaraproject/IdentityAndConsentManagement/issues/82, and move on to the next needed discussion without blocking this track by creating new dependencies between issues
```